### ISSUE TITLE
Editpost return error "invalid postid"

### ISSUE THREAD

**Issue by veryrobin** [alexjamesbrown/JoeBlogs#26]
Editpost return error "invalid postid"!!! @wordpress 3.5.1

**Comment by alexjamesbrown** [comment]
Do you have any more detail?

**Comment by veryrobin** [comment]
The error detail info same as "https://github.com/alexjamesbrown/JoeBlogs/issues/21"

**Comment by veryrobin** [comment]
I think I found the problem.

The JoeBlogs.XmlRpcInterfaces.IWordPressXmlRpc 86 Line code:
[XmlRpcMethod("wp.editPost")]
bool EditPost(int postid, string username, string password, XmlRpcPost content, bool publish);

The wordpress  wp.editPost api :
Parameters
int blog_id
string username
string password
int post_id
struct content: See #wp.newPost for valid set of fields. Only needs to contain fields that you wish to modify; all other fields will retain their current values.

The JoeBlogs.WordPressWrapper Editpost function code:
 public virtual bool EditPost(int postID, Post content, bool publish)
{
            return _wrapper.EditPost(postID, Username, Password, Map.From.Post(content), publish);
 }

So:parameters error. don't have blog_id and post_id position error。

**Comment by veryrobin** [comment]
Yeah,I got it! Modify The JoeBlogs.XmlRpcInterfaces.IWordPressXmlRpc 86 Line code:"[XmlRpcMethod("wp.editPost")]" to [XmlRpcMethod("metaWeblog.editPost")].

**Comment by alexjamesbrown** [comment]
Are you able to commit this into the git repository as a a pull request?

**Comment by veryrobin** [comment]
I try it........

**Comment by simon-duvall** [comment]
Actually, looks like metaWeblog.editPost was never implemented.
